features/providers/non_passported_journey.feature

```diff
@@ -65,9 +66,28 @@ Feature: Non-passported applicant journeys
     And I click 'Save and continue'
 
     Then I should be on the 'income_summary' page showing "Sort your client's income into categories"
+    And the following sections should exist:
+      | tag | section |
+      | h2  | Benefits |
+      | h2  | Disregarded benefits |
+      | h2  | Financial help from friends or family |
+
+    And the following sections should not exist:
+      | tag | section |
+      | h2  | Housing benefits |
+
     Then I click the first link 'View statements and add transactions'
     Then I select the first checkbox
     And I click 'Save and continue'
+
+    # Then I click the '2nd' link 'View statements and add transactions'
+    # Then I select the first checkbox
+    # And I click 'Save and continue'
```

Comment:
I want to uncomment this, but I'm not sure how to set up the appropriate VCR cassette so that state benefit data is returned 🤔

Reply:
could pair on tomorrow if you like, for vcr i generally ensure the local machine config is pointing at relevant staging remote apps then delete the exist cassette and rerun feature test.

Reply:
Sounds good, I'll try and schedule something. 

I don't like Cucumber or VCR so it's like two nemeses colliding 😆